crates/net/src/gossip/driver.rs

```diff
@@ -66,8 +67,11 @@ impl GossipDriver {
             message,
         })) = event
         {
+            tracing::debug!("Received message with topic: {}", message.topic);
```

Comment:
pedantic nit: in (almost) all the codebase we import tracing at the top level, would be good to do it for consistency